sova/config/loader.py

```
@@ -21,36 +23,35 @@
+        toml_flat = _flatten_toml(data)
+        _migrate_deprecated_keys(toml_flat)
 
-    # Default config
-    return ProjectConfig()
+    from sova.config.db_loader import _try_load_from_db
 
+    db_overrides = _try_load_from_db(project_dir)
+    if db_overrides:
+        merged = _deep_merge(toml_flat, db_overrides)
+    else:
+        merged = toml_flat
```

Ruff (0.16.1)

[warning] Use ternary operator `merged = _deep_merge(toml_flat, db_overrides) if db_overrides else toml_flat` instead of `if`-`else`-block

Replace `if`-`else`-block with `merged = _deep_merge(toml_flat, db_overrides) if db_overrides else toml_flat`

(SIM108)